modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala

```diff
@@ -95,6 +95,7 @@ object Model {
   object SequenceState {
     case object Completed         extends SequenceState
     case object Running           extends SequenceState
+    case object Stopping          extends SequenceState
```

Comment:
😕 We had some discussions while you were away about banning the word `Stop` from the code and rename it all to `Pause`. In any case if this doesn't show up in the UI, I think we'll be fine.